packages/two_dimensional_scrollables/CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
 ## NEXT
```

Comment:
Same here.

Reply:
Same applies here; the changes aren't to lib/main.dart.